content/docs/development.md

```diff
@@ -316,18 +370,70 @@ Please note that you need to have the `oc` binary available in your
 The goal here is to download the project to your laptop/PC, and to
 install Minishift, the VM that contains OpenShift.
 
-    $ git pull https://github.com/syndesisio/syndesis.git # or own fork
-    $ cd syndesis
-    
-    # install minishift
-    $ syndesis minishift --full-reset --project syndesis --maven-mirror 
-    
-    # install syndesis with jaeger addon
-    $ syndesis minishift --install --project syndesis --nodev --app-options " --addons jaeger"
+```shell
+$ git pull https://github.com/syndesisio/syndesis.git # or own fork
+$ cd syndesis
+
+# install minishift
+$ syndesis minishift --full-reset --project syndesis --maven-mirror
+```
+
+You can also include other options when setting up, for example
+including an addon:
+
+```shell
+# install syndesis with jaeger addon
+$ syndesis minishift --install --project syndesis --nodev --app-options " --addons jaeger"
+```
+<div class="alert alert-info admonition" role="alert"> <i class="fa
+important"></i> The jaeger is installed to support the activity log
+backed by jaeger opentracing, which is faster than the activity log
+stored in postgresql.  </div>
 
-<div class="alert alert-info admonition" role="alert">
-  <i class="fa important"></i> The jaeger is installed to support the activity log backed by jaeger opentracing, which is faster than the activity log stored in postgresql.
-</div>
+### Running without Kubernetes/OpenShift
+
+For some development tasks you can run Syndesis without running on a
+Kubernetes/OpenShift cluster. You won't be able to publish integrations,
+see activities or gather metrics, or any other functionality that
+requires a running cluster. This is best suited for quick turnaround
+development of backend APIs together with UI features. You'll need three
+components running the Syndesis backend, the PostgreSQL database it uses
+and the UI.
+
+Start by running a PostgreSQL database in a Docker container:
+
+```shell
+$ docker run -d --rm -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=password -e POSTGRES_DB=syndesis postgres
+```
+<div class="alert alert-info admonition" role="alert"> <i class="fa
+important"></i> the `--rm` option will remove the container once done
+and all the stored data in the database with it, if you wish to keep the
+data omit that option </div>
+
+Next run the Syndesis backend, this step requires that you have
+downloaded all the dependencies and built the backend:
+
+```shell
+$ syndesis build -f -c --backend # to download dependencies and build the backend
+$ (cd app && ./mvnw -f server/runtime spring-boot:run)
+```
+
+The backend is started once you see the a line containing this in the
```

Comment:
typo here, though I'm sure this was pre-existing "see the a line" should be "see a line"